core/commands/validate.go

```diff
@@ -0,0 +1,25 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Date Command
+func ValidateCommand() (*cobra.Command, error) {
+	// creating new command
+	// using localizer to access default text by ID provided in locals
+	cmd := &cobra.Command{
+		Use:     "validate",
+		Short:   "validate custom commands",
+		Long:    "validates the custom commands with the defined rules",
+		Example: "host charmil validate",
+		Args:    cobra.ExactArgs(0),
+		// SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			println("validate")
```

Comment:
Should something be here?

Reply:
Yes this is just a proof of concept to get the charmil commands working.

Reply:
Yes.. but docs are actually quite good example how things might work. If we split we can merge docs and leave validation POC separate.